### ISSUE TITLE
Can't copy sequence from gff3 features

### ISSUE THREAD

**Issue by nhartwic** [GMOD/jbrowse-components#2950]
I have a jbrowse2 instance setup with a variety of genomes and gff3 files containing gene models. Whenever I click on a gene, the area that should populate cds/protein/exon/whatever sequences just says "Loading gene sequence..." and never populates. As a result, I can't copy sequences using either button.

Example session that should show the issue:
http://salk-tm-pub.s3-website-us-west-2.amazonaws.com/jbrowse2/?session=share-vyIMJG5-iF&password=001Vr

Jbrowse2 has no trouble pulling sequence from the fasta, as demonstrated by zooming in and enabling the sequence track or by highlighting a region and right clicking and "get sequence". Jbrowse2 seems to be parsing the gene models correctly based on the fact that its drawing reasonable models. 

Any help would be appreciated

**Comment by nhartwic** [comment]
Upgrading from 1.7.6 to 1.7.7 seems to have solved the issue.

**Comment by rbuels** [comment]
This is strange, because I don't see any changes in 1.7.7 vs 1.7.6 that would have affected this. Could this have been related to browser caching?

**Comment by cmdcolin** [comment]
It is briefly mentioned in the changelog here https://jbrowse.org/jb2/blog/2022/05/04/v1.7.7-release/

The CoreGetFeatures API's ref renaming was modified in 1.7.6 which caused it to freeze if no assembly name was supplied, and then fixed in 1.7.7

**Comment by nhartwic** [comment]
@rbuels 

I cant see how. I rebuilt a completely fresh local instance on 1.7.6 using one of my fasta/gff3 that wasn't working and even those local instances weren't populating the sequence area for features. But the second I ran "jbrowse upgrade", it worked, and repeating this sollution on our public instance resolved the issue.

@cmdcolin 

Makes sense. Thanks for the information. And thanks for fixing the bug so promptly.
